star-dash/star-dash/Persistence/LevelData.swift

```diff
@@ -0,0 +1,21 @@
+//
+//  LevelData.swift
+//  star-dash
+//
+//  Created by Lau Rui han on 22/3/24.
+//
+
+import Foundation
+
+struct LevelData: Encodable, Decodable {
+    var id: Int64
+    var name: String
+    var entities: [EntityPersistable]
+
+    init(id: Int64, name: String, entities: [EntityPersistable]) {
+        self.id = id
+        self.name = name
+        self.entities = entities
+    }
+
+}
```

Comment:
Maybe can group `LevelData` and `data.json` into 1 folder for organization, since they both only deal with preloaded levels.

Reply:
changed